src/main/java/com/twilio/rest/api/v2010/account/MessageReader.java

```diff
@@ -95,31 +98,23 @@ public MessageReader setFrom(final String from) {
      * sent on or before midnight on a date, and `DateSent&gt;=YYYY-MM-DD` to read
      * messages sent on or after midnight on a date..
      *
-     * @param absoluteDateSent Filter by date sent
+     * @param dateSent Filter by date sent
      * @return this
      */
-    public MessageReader setDateSent(final DateTime absoluteDateSent) {
-        this.rangeDateSent = null;
-        this.absoluteDateSent = absoluteDateSent;
+    public MessageReader setDateSent(final LocalDate dateSent) {
```

Comment:
Why is this a `LocalDate` type when it previously had date_time precision?

Reply:
Great catch! I'm going to double check all of these mappings. This particular one should be a `ZonedDateTime`.  

Reference:
`org.joda.time.DateTime ~> java.time.ZonedDateTime`
`org.joda.time.LocalDate ~> java.time.LocalDate`

Reply:
Are you making use of the type mappings defined in api-defs? https://code.hq.twilio.com/twilio/api-definitions/blob/master/types/java.json

Reply:
Yes: https://code.hq.twilio.com/twilio/api-definitions/pull/2068/files

This will also be updated as part of this PR to remove the Range type.